common/app/com/commercetools/sunrise/play/SunriseHttpFilters.java

```diff
@@ -1,5 +1,7 @@
-package com.commercetools.sunrise.common.basicauth;
+package com.commercetools.sunrise.play;
```

Comment:
:D so we use different strategies, I used packages like play would have like, so I would have put it into a package filters.

play is too general and will contain finally a lot of classes with very different types and purposes